xml/System.Runtime.InteropServices/DllImportAttribute.xml

```diff
@@ -488,7 +488,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Set the <xref:System.Runtime.InteropServices.DllImportAttribute.PreserveSig> field to `true` to translate unmanaged signatures with `HRESULT` values directly; set it to `false` to automatically convert `HRESULT` return values to exceptions. By default, the <xref:System.Runtime.InteropServices.DllImportAttribute.PreserveSig> field is `true`.  
+ Set the <xref:System.Runtime.InteropServices.DllImportAttribute.PreserveSig> field to `true` to translate unmanaged signatures with `HRESULT` values directly; set it to `false` to automatically convert `HRESULT` return values to exceptions. By default, the <xref:System.Runtime.InteropServices.DllImportAttribute.PreserveSig> field is `false`.  
```

Comment:
I do not think that this is change is right.

This field is `true` by default when one is using this attribute. For example, if you write `[DllImport("my.dll")]`, it will result into `PreserveSig` set to true.